lib/library/books/books.ex

```diff
@@ -19,9 +19,34 @@ defmodule Library.Books do
 
   """
   def list_books do
-    Repo.all(Book)
+    query = from b in Book,
```

Comment:
can the queries all be in one place? like [books.ex](https://github.com/dwyl/library/blob/master/lib/library/books/books.ex). It makes it easier to avoid repetition if all the queries are listed together.